File: Ruby/fizzbuzz.rb
def fizzbuzz(num)
  n = 1
  while  n <= num
    if n % 5 == 0 && n % 3 == 0
      puts "fizzbuzz"
    elsif n % 5 == 0
      puts "buzz"
    elsif n % 3 == 0
      puts "fizz"
    else
      puts n
    end
    n += 1
  end
end

fizzbuzz(100)

File: Python/fizzbuzz.py
num = 100
n = 1

while n <= num:
    if n % 15 == 0:
        print("fizzbuzz")
    elif n % 5 == 0:
        print("buzz")
    elif n % 3 == 0:
        print("fizz")
    else:
        print(n)
    n += 1